src/javascript/binary/static/profittable.js

```diff
@@ -0,0 +1,14 @@
+
+pjax_config_page("profit_tablews", function(){
+    return {
+        onLoad: function() {
+            Content.populate();
```

Comment:
IMO, we should split all text into their corresponding domain (statement, trade, profit table)
so that we dont share the localize object? since all text is in text.json after all